### PR TITLE
Add dashboard user doc

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add dashboard user doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7187
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
I've finished the self-check of this PR. PTAL. Thanks.

File: dashboard/dashboard-user.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+>
+> Users with high privileges such as `ALL PRIVILEGES` or `SUPER` can still sign in with TiDB Dashboard. Therefore, to comply with the least privilege principle, it is highly recommended that you create users with the preceding specific privileges to prevent unintended operations. See [Privilege Management](/privilege-management.md) for more information on these privileges.
+
+When the SQL user does not meet the preceding privilege requirements, it fails to sign in with TiDB Dashboard, as shown below.
+
+![insufficient-privileges](/media/dashboard/dashboard-user-insufficient-privileges.png)
+
+## Example: Create a least-privileged SQL user to access TiDB Dashboard
+
+Wen [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is not enabled on the connected TiDB server, you can create a SQL user `dashboardAdmin` that can sign in with TiDB Dashboard by executing the following SQL statement:
```
Comment (shichun-0415) [inline dashboard/dashboard-user.md:46]:
```suggestion
When [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is not enabled on the connected TiDB server, you can create a SQL user `dashboardAdmin` that can sign in with TiDB Dashboard by executing the following SQL statement:
```

File: dashboard/dashboard-access.md [hunk 2]
```diff
@@ -28,7 +28,10 @@ You can use TiDB Dashboard in the following common desktop browsers of a relativ
 
 ## Sign in
 
-For the first-time access, TiDB Dashboard displays the user sign in interface, as shown in the image below. You can sign in using the TiDB `root` account. By default, the `root` password is empty.
+After access to TiDB Dashboard, you will be directed to the user login interface, as shown in the image below.
+
+You can sign in with TiDB Dashboard using the TiDB `root` account. By default, the `root` password is empty.
+If you have created a [User-defined SQL User](/dashboard/dashboard-user.md), you can sign in by using this account and the corresponding password.
```
Comment (baurine) [inline dashboard/dashboard-access.md:34]: Compared with the Chinese version, here seems to miss the list symbol?

File: dashboard/dashboard-ops-security.md [hunk 3]
```diff
@@ -8,11 +8,20 @@ aliases: ['/docs/dev/dashboard/dashboard-ops-security/']
...
 
-## Set a strong password for TiDB `root` user
+## Enhance security of TiDB users
+
+### Set a strong password for TiDB `root` user
 
 The account system of TiDB Dashboard is consistent with that of TiDB SQL users. By default, TiDB's `root` user has no password, so accessing TiDB Dashboard does not require password authentication, which will give the malicious visitor high privileges, including executing privileged SQL statements.
 
-It is recommended that you set a strong password for TiDB `root` user. See [TiDB User Account Management](/user-account-management.md) for details.
+It is recommended that you set a strong password for TiDB `root` user. See [TiDB User Account Management](/user-account-management.md) for details. Alternatively, you can disable TiDB `root` user.
```
Comment (baurine) [inline dashboard/dashboard-ops-security.md:17]: Compared with the Chinese version, here seems to miss the list symbol?

File: dashboard/dashboard-user.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    GRANT RESTRICTED_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+```
+
+To modify the configuration items on the interface after signing in with TiDB Dashboard, the user-defined SQL user must be granted with the following privilege:
+
+```
+GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+```
+
+Sign in with TiDB Dashboard
```
Comment (baurine) [inline dashboard/dashboard-user.md:79]: Compared with the Chinese version, it seems to miss the secondary heading format?

File: dashboard/dashboard-user.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+> **Note:**
+>
+> Users with high privileges such as `ALL PRIVILEGES` or `SUPER` can still sign in with TiDB Dashboard. Therefore, to comply with the least privilege principle, it is highly recommended that you create users with the preceding specific privileges to prevent unintended operations. See [Privilege Management](/privilege-management.md) for more information on these privileges.
+
+When the SQL user does not meet the preceding privilege requirements, it fails to sign in with TiDB Dashboard, as shown below.
+
+![insufficient-privileges](/media/dashboard/dashboard-user-insufficient-privileges.png)
+
+## Example: Create a least-privileged SQL user to access TiDB Dashboard
```
Comment (baurine) [inline dashboard/dashboard-user.md:44]: It seems this part content format is not exactly the same as the Chinese version.

**Review by baurine — APPROVED** [review]

File: dashboard/dashboard-user.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    ```sql
+    CREATE USER 'dashboardAdmin'@'%' IDENTIFIED BY '<YOUR_PASSWORD>';
+    GRANT PROCESS, CONFIG ON *.* TO 'dashboardAdmin'@'%';
+    GRANT SHOW DATABASES ON *.* TO 'dashboardAdmin'@'%';
+    GRANT DASHBOARD_CLIENT ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+    -- To modify the configuration items on the interface after signing in with TiDB Dashboard, the user-defined SQL user must be granted with the following privilege:
+
+    ```
```
Comment (qiancai) [inline dashboard/dashboard-user.md:57]:
```suggestion
    ```sql
```

File: dashboard/dashboard-user.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+- When [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is not enabled on the connected TiDB server, you can create a SQL user `dashboardAdmin` that can sign in with TiDB Dashboard by executing the following SQL statement:
+
+    ```sql
+    CREATE USER 'dashboardAdmin'@'%' IDENTIFIED BY '<YOUR_PASSWORD>';
+    GRANT PROCESS, CONFIG ON *.* TO 'dashboardAdmin'@'%';
+    GRANT SHOW DATABASES ON *.* TO 'dashboardAdmin'@'%';
+    GRANT DASHBOARD_CLIENT ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+    -- To modify the configuration items on the interface after signing in with TiDB Dashboard, the user-defined SQL user must be granted with the following privilege:
```
Comment (qiancai) [inline dashboard/dashboard-user.md:55]: -- 代表 SQL 语句里的单行注释，如果把这句话从 SQL 语句里单独拿出来了，-- 就不需要了

File: dashboard/dashboard-user.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    CREATE USER 'dashboardAdmin'@'%' IDENTIFIED BY '<YOUR_PASSWORD>';
+    GRANT PROCESS, CONFIG ON *.* TO 'dashboardAdmin'@'%';
+    GRANT SHOW DATABASES ON *.* TO 'dashboardAdmin'@'%';
+    GRANT DASHBOARD_CLIENT ON *.* TO 'dashboardAdmin'@'%';
+    GRANT RESTRICTED_STATUS_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    GRANT RESTRICTED_TABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    GRANT RESTRICTED_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+    -- To modify the configuration items on the interface after signing in with TiDB Dashboard, the user-defined SQL user must be granted with the following privilege:
```
Comment (qiancai) [inline dashboard/dashboard-user.md:73]: 同上

File: dashboard/dashboard-user.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+    GRANT DASHBOARD_CLIENT ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+    -- To modify the configuration items on the interface after signing in with TiDB Dashboard, the user-defined SQL user must be granted with the following privilege:
+
+    ```
+    GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+- When [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is enabled on the connected TiDB server, you can create a SQL user `dashboardAdmin` that can sign in with TiDB Dashboard by executing the following SQL statement:
```
Comment (qiancai) [inline dashboard/dashboard-user.md:61]:
```suggestion
- When [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is enabled on the connected TiDB server, to create a SQL user `dashboardAdmin` that can sign in with TiDB Dashboard, execute the following SQL statements:
```

File: dashboard/dashboard-user.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+    GRANT SHOW DATABASES ON *.* TO 'dashboardAdmin'@'%';
+    GRANT DASHBOARD_CLIENT ON *.* TO 'dashboardAdmin'@'%';
+    GRANT RESTRICTED_STATUS_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    GRANT RESTRICTED_TABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    GRANT RESTRICTED_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+    -- To modify the configuration items on the interface after signing in with TiDB Dashboard, the user-defined SQL user must be granted with the following privilege:
+
+    ```
```
Comment (qiancai) [inline dashboard/dashboard-user.md:75]:
```suggestion
    ```sql
```

File: dashboard/dashboard-user.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+    GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    ```
+
+## Sign in with TiDB Dashboard
+
+After creating a SQL user that meets the privilege requirements of TiDB Dashboard, you can [Sign in](/dashboard/dashboard-access.md#Sign in) with TiDB Dashboard by using this user.
```
Comment (qiancai) [inline dashboard/dashboard-user.md:81]:
```suggestion
After creating a SQL user that meets the privilege requirements of TiDB Dashboard, you can use this user to [Sign in](/dashboard/dashboard-access.md#Sign in) with TiDB Dashboard.
```
```suggestion
After creating a SQL user that meets the privilege requirements of TiDB Dashboard, you can [Sign in](/dashboard/dashboard-access.md#Sign in) with TiDB Dashboard by using this user.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**